apps/storybook/src/forms/Textarea.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Button, Flex, Text, Textarea } from "@optiaxiom/react";
+
+const meta: Meta<typeof Textarea> = {
+  component: Textarea,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Textarea>;
+
+export const Default: Story = {
+  args: {
+    placeholder: "Enter text....",
+  },
+};
+
+export const WithTopSection: Story = {
+  args: {
+    placeholder: "Enter text....",
+    topSection: <Text>Top Section</Text>,
+  },
+};
+
+export const WithBottomSection: Story = {
+  args: {
+    bottomSection: <Text>Bottom Section</Text>,
+    placeholder: "Enter text....",
+  },
+};
+
+export const Disabled: Story = {
+  args: {
+    disabled: true,
+    placeholder: "Enter text....",
+  },
+};
+
+export const Readonly: Story = {
+  args: {
+    placeholder: "Enter text....",
+    readOnly: true,
+  },
+};
+
+export const NoResize: Story = {
+  args: {
+    placeholder: "Enter text....",
+    resize: "none",
+  },
+};
+
+export const Custom: Story = {
```

Comment:
can we use a better name here? or maybe make it more like the opal design - or remove it since the other stories already cover the sections

Reply:
gotcha bhai